usr/share/rear/build/GNU/Linux/100_copy_as_is.sh

```diff
@@ -112,7 +112,7 @@ done >$copy_as_is_exclude_file
 # cf. https://github.com/rear/rear/pull/2405#issuecomment-633512932
 # FIXME: The following code fails if file names contain characters from IFS (e.g. blanks),
 # cf. https://github.com/rear/rear/issues/1372
-if ! tar -v -X $copy_as_is_exclude_file -P -C / -c ${COPY_AS_IS[*]} 2>$copy_as_is_filelist_file | tar $v -C $ROOTFS_DIR/ -x 1>/dev/null ; then
+if ! tar -v -X $copy_as_is_exclude_file -P -C / -c ${COPY_AS_IS[*]} 2>$copy_as_is_filelist_file | tar $v -C $ROOTFS_DIR/ -x -i 1>/dev/null ; then
```

Comment:
@rmetrich 
please enhance the comment before that 'tar' command
to explain the reason behind why '-i' is needed, cf.
https://github.com/rear/rear/wiki/Coding-Style#code-must-be-easy-to-understand-answer-the-why